### ISSUE TITLE
cd does not execute

### ISSUE THREAD

**Issue by irapha** [iRapha/same-tbh#10]
```
$ tbh 
tbh (recording) $ ls
README.md   same        tbh
tbh (recording) $ cd ..
tbh (recording) $ ls
README.md   same        tbh
```

**Comment by joshuamorton** [comment]
since everything is occurring in a subprocess that won't work, you may need to fudge and have os.chdir execute in those cases, although IDK if that'll even work generally.

Or you can have all of the commands execute in the _same_ subprocess.

**Comment by irapha** [comment]
Yeah. I think we will need to create a sub process that will last for the entire recording/execution session. Right now, I'm just making `os.system(command)` calls
